test/jdk/java/net/Inet4Address/PingThis.java

```diff
@@ -45,7 +45,12 @@
 
 public class PingThis {
     public static void main(String args[]) throws Exception {
-        if (System.getProperty("os.name").startsWith("Windows")) {
+        osname = System.getProperty("os.name")
+        /*
```

Comment:
Doesn't look like the line above will compile.

Reply:
Good catch!

Reply:
too much Python (no semi-colons) - great catch.

Reply:
@aixtools, if you enable GitHub Actions on your forked repo, it should then automatically trigger a GitHub Actions job for your changes in your PR and those results/run will be visible in this PR. The openjdk/jdk repo has been configured to run `tier1` build/test on PR submission, so errors like this are easily caught. Of course, if you build this locally, that's much quicker to catch too.

Reply:
This is a test in tier2 though - so I don't expect it will be caught by github actions

Reply:
re: actions - the default seems to be enable all, so unclear on what I would need to do extra to get these working.
Guess I need to study how to build locally, as well as how to use the 'custom' variation of a Grinder (jenkins) test.

(I'm just an `admin`, lets say the 'ops` side of `devops` - so I have many things to learn re: the `dev` side of things).

Reply:
Saying resolved - after using @requires rather than check in code - removing the windows code check as well